apis/infrastructure/v1/nim_types.go

```diff
@@ -0,0 +1,12 @@
+package v1
+
+import (
+	operatorv1 "github.com/openshift/api/operator/v1"
+)
+
+// nimSpec enables Nvidida's NIM integration
+type NimSpec struct {
+	// +kubebuilder:validation:Enum=Managed;Removed
+	// +kubebuilder:default=Managed
```

Comment:
should this by default to `Managed` or `Removed`?

Reply:
@zdtsw should be removed by default.. I updated to reflect this.

Reply:
not sure if this has been updated or got reverted back to Managed.
i guess this is something from requirement